yarn.lock

```diff
@@ -3371,7 +3376,7 @@
     "@polkadot/util" "^6.3.1"
     bn.js "^4.11.9"
 
-"@polkadot/types@4.2.1":
+"@polkadot/types@4.2.1", "@polkadot/types@4.7.2":
```

Comment:
Ah so this is why 4.7.2 was being fetched and was putting a demand on nodejs 14 :)
Al though things seem to be working well, yes it might be prudent.

Reply:
done in aaeb89561a336ba2303110b1257d183edb11cdb0